readme.txt

```diff
@@ -74,14 +77,15 @@ If you want to do something more, you can consider:
   complicated an image is). This is not a payment for a campaign; this is to
   fund new portraits. In other words, this is in order to allow the others
   enjoy the campaign even more. And we may then decide to create a sequel :)
-  Use moneybookers or paypal. You have to register, though. The transfer with
-  moneybookers costs about 1 euro (0.5 fee by moneybookers, but in my country
-  my bank also charges for transferring to my account in moneybookers); 
-  http://www.howtopayonline.org/moneybookers.html. Paypal charges percentage of
-  the amount, I *think* it is about 5%. My account is "szopen@europe.com" -- if
-  you made a transfer, you may also consider to send a suggestion for which
-  portrait should be done next. TODO: Sigil's credit card or account here as
-  an alternative payment form for a donation?
+  Use PayPal. You have to register, though. PayPal charges a percentage of
+  the amount, I *think* it is about 5%. It used to be that the suggested
+  donation account for this campaign on PayPal was szopen's, but he has
+  suggested to update that to an account for someone more currently involved
+  with maintainence of the campaign, so mine (egallager's) is here:
+  https://www.paypal.com/paypalme/egallager
+  I am not sure if Yumi/nemaara has one; I'll have to ask her.
+  You can also sponsor egallager via the GitHub Sponsors program here:
+  https://github.com/sponsors/cooljeanius

```

Comment:
I'm wondering, if it's ok to add my GitHub Sponsors link here, would it be ok to also add a `.github/FUNDING.yml` to this repo? szopen suggested changing this line of the README, but he didn't say anything about a `FUNDING.yml` though...

Reply:
You can do either one, realistically this campaign isn't going to get any funding in the near future unless wesnoth takes off and we start making some really big changes though.

Reply:
ok, `FUNDING.yml` added in 96981c6